code/script.js

```diff
@@ -1,15 +1,72 @@
-// Variables that point to selected DOM elements
-const chat = document.getElementById('chat')
+"use strict"
+// VARIABLES THAT POINT TO SELECTED DOM ELEMENTS
+const chat = document.getElementById('chat');
+const form = document.getElementById('name-form');
+const submitBtn = document.getElementById('submit-btn');
+const inputField = document.getElementById('name-input');
+let soundEffect = new Audio('assets/pop.mp3'); // Sound effect to the chat
 
-// If you need any global variables that you can use across different functions, declare them here:
+// GLOBAL VARIABLES
+let userAnswer = ""; // stores users answer in a variable globally for use anywhere
+let selectedGenre = ""; // stores selected genre in a variable globally for use anywhere
 
+// GLOBAL OBJECTS
+// Stores the different movie-choices in an object for easier handling further down.
```

Comment:
Great job using the arrays! Not sure on how and guessing here, but is there a way of making this code (even) more great by merging the oldies and newbie in one array? An array in an array/nested somehow? :)

Reply:
You're right!! That would probably make much more sense.. Perhaps I was a bit rusty on working with objects at this point, or just tired over all 😅 Thanks for the tip!